tribestream-api-registry-webapp/src/main/static/assets/styles/_component_markdown.sass

```diff
@@ -49,11 +49,15 @@ div.markdown-help-content
     bottom: $distance
     right: $distance
     left: $distance
-    padding: 40px
+    padding: 40px 50px 40px 50px
     overflow: auto
     background-color: white
     @include code-style
-
+    >i.fa.fa-times-circle
+      position: fixed
+      top: 55px
+      right: 65px
```

Comment:
Does not look like its aligned. Change it to 55px to get proper alignement. 

![screen shot 2016-11-21 at 9 17 55 am](https://cloud.githubusercontent.com/assets/1918442/20485912/ae05c9e2-afcb-11e6-923f-b851200aebec.png)

![screen shot 2016-11-21 at 9 18 44 am](https://cloud.githubusercontent.com/assets/1918442/20485936/c09f09ce-afcb-11e6-907b-7f133eea5a0c.png)